### DISCUSSION TITLE
New theme for documentation site

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1628] [Team Posts]
Now [Sympa site](https://sympa-community.github.io/) puts a familiar skin.

  - Modify [sympa-community/sympa-pages-theme](/sympa-community/sympa-pages-theme) to correct any errors on the skin (default branch is `main`).

**Comment by racke** [comment]
Great, that's definitely a big improvement!

**Comment by ikedas** [comment]
About GitHub Pages feature please read: ~~https://docs.github.com/pages~~ https://docs.github.com/pages/getting-started-with-github-pages

Currently these repos are published with Pages:
  - [sympa-community.github.io](/sympa-community/sympa-community.github.io)
  - [sympa](/sympa-community/sympa) (Index is README.md)
  - [gpldoc](/sympa-community/gpldoc)

I think it'd be better to publish [sympa-contribs](/sympa-community/sympa-contribs) (README.md files may be used as index).  A problem is that Pages can't recognize the files without extension as text files.  @ldidry , can you rename `LICENSE` to `LICENSE.md` or such?

EDIT: Addition: [assets](/sympa-community/assets).

**Comment by ikedas** [comment]
[sympa-contribs](/sympa-community/sympa-contribs) was published too.  URL hierarchy is https://sympa-community.github.io/sympa-contribs

**Reply by ldidry** [reply]
Sorry, I didn’t saw your demand before right now. I renamed the LICENSE files to LICENSE.md.
